packages/react/src/file-input/docs/overview.mdx

```diff
@@ -70,3 +71,40 @@ Use the `invalid` prop to indicate if the user input is invalid.
 ```jsx live
 <FileInput label="Invalid" invalid message="This input is invalid" />
 ```
+
+## Accept
+
+Use the `accept` prop to guide the user toward the accepted file types. This will automatically render a hint, although you can override this by providing your own `hint`.
+
```

Comment:
I think it's probably worth talking about the difference between the array version and the benefits it can provide "All image files" etc

Reply:
Just added this.